app/assets/stylesheets/downtime.scss

```diff
@@ -7,3 +7,18 @@
     margin-right: $default-horizontal-margin;
   }
 }
+
+p {
+  font-weight: bold;
+  font-size: 16px;
+}
```

Comment:
Sorry I'm late to the party here. I was going to just click merge but this caught my eye. Won't this set the styling of paragraphs for the entire app?

Reply:
Good spot. Have pushed a fix